api/nodes/views.py

```diff
@@ -130,34 +131,25 @@ def get_draft(self, draft_id=None):
 
 class WaterButlerMixin(object):
 
-    path_lookup_url_kwarg = 'path'
```

Comment:
You should be careful about refactoring things like this, especially on a PR that is as large as this one. This is the kind of thing that leads to merge conflicts, so the potential for trouble is a lot greater than the gain.